cmd/gapit/screenshot.go

```diff
@@ -174,3 +182,25 @@ func (verb *screenshotVerb) frameCommand(ctx context.Context, capture *path.Capt
 	fmt.Printf("Frame Command: %v\n", eofEvents[verb.Frame].Command.GetIndices())
 	return eofEvents[verb.Frame].Command, nil
 }
+
+func rescaleBytes(ctx context.Context, data []byte, max int) {
```

Comment:
Have you thought about just rescaling dynamically?
(0->Max) -> (0->1)

Reply:
Could be done as a follow-up but it would save some having to type.
That being said, someone may want to avoid rescaling as well.

Reply:
Never mind! I misread this, you are already doing what I had said.
Ignore me